src/Functions/argslen.js

```diff
@@ -0,0 +1,3 @@
+export default function (msg) {
+  return msg.content.split(' ').length;
```

Comment:
split in /\s+/ so it won't consider a message like 'a   a' with 3 args (['a', ' ', 'a'])